src/xdocs/config_misc.xml

```diff
@@ -2490,10 +2490,40 @@ public record MyRecord1 {
           To configure the check:
         </p>
         <source>
+&lt;module name=&quot;UniqueProperties&quot;/&gt;
+        </source>
+        <p>
+          Example: in foo.properties file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // violation
+        </source>
+        <p>
+          To configure the check to scan custom file extensions:
+        </p>
+        <source>
 &lt;module name=&quot;UniqueProperties&quot;&gt;
-  &lt;property name=&quot;fileExtensions&quot; value=&quot;properties&quot; /&gt;
+  &lt;property name=&quot;fileExtensions&quot; value="PROPERTIES"/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example: in foo.PROPERTIES file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // violation
+        </source>
+        <p>
+          Example: in foo.properties file
+        </p>
+        <source>
+key.one=44
+key.two=32 // OK
+key.one=54 // OK, file is not checked
```

Comment:
This can be confusing for Windows users who know that `.properties` and `. PROPERTIES` are the same file extension. I understand that the reason is not to pollute the spell checker with new words. Maybe the `.custom` extension would be fine here?

Reply:
or even `.customProperties`, since the spell checker is aware of camelCase words.

Reply:
Okay, so I should change `PROPERTIES` to `customProperties` for the second example. This would look as foll:
```
<module name="UniqueProperties">
  <property name="fileExtensions" value="customProperties"/>
</module>
```
Example: in foo.customProperties file
```
key.one=44
key.two=32 // OK
key.one=54 // violation
```
Example: in foo.properties file
```
key.one=44
key.two=32 // OK
key.one=54 // OK, file is not checked
```
Let me know if I should change this?
Thanks for the suggests!

Reply:
> Let me know if I should change this?

Yes, this should pass the spell checker.